articles/article-093-rices-theorem.md

```diff
@@ -0,0 +1,40 @@
+<!-- BEGIN TITLE -->
+# Why There Is No Perfect Anti-Virus
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In computer science, simple "Yes/No" questions are called **decision
+problems**.  A decision problem is called **decidable** if we can
+write a program that can always find an answer for such a problem.
+
+An example for such a decision problem is "does this program contain a
+computer virus".  If this problem was decidable, there could be
+another program that could always correctly answer this question with
+"Yes" or "No".  A mathematician called Henry Gordon Rice showed that
```

Comment:
with a "yes" or "no". 
You do not need to capitalize the yes and no (you can make the change above if you want).

Reply:
Agreed